report/src/main/java/com/hartwig/actin/report/pdf/tables/molecular/RecentMolecularSummaryGenerator.java

```diff
@@ -36,113 +37,162 @@ public class RecentMolecularSummaryGenerator implements TableGenerator {
+    private Optional<String> genesWithHighDriverDeletionStringOption() {
+        return summaryStringOptionForGeneAlterations(molecular.drivers()
+                .copyNumbers()
+                .stream()
+                .filter(copyNumber -> copyNumber.type().isLoss() && !copyNumber.type().isGain()));
     }
 
     @NotNull
-    private static String concat(@NotNull Set<String> strings) {
-        StringJoiner joiner = Formats.commaJoiner();
-        for (String string : strings) {
-            joiner.add(string);
-        }
-        return Formats.valueOrDefault(joiner.toString(), "None");
+    private Optional<String> genesWithHighDriverHomozygousDisruptionStringOption() {
+        return summaryStringOptionForGeneAlterations(molecular.drivers().homozygousDisruptions().stream());
+    }
+
+    @NotNull
+    private Optional<String> highDriverGeneFusionsStringOption() {
+        String fusions = molecular.drivers()
+                .fusions()
+                .stream()
+                .filter(fusion -> fusion.driverLikelihood() == DriverLikelihood.HIGH)
+                .map(Driver::event)
+                .collect(Collectors.joining(", "));
+        return Optional.of(fusions.isEmpty() ? Formats.VALUE_NONE : fusions);
+    }
+
+    @NotNull
+    private Optional<String> highDriverVirusDetectionsStringOption() {
+        String fusions = molecular.drivers()
+                .viruses()
+                .stream()
+                .filter(virus -> virus.driverLikelihood() == DriverLikelihood.HIGH)
+                .map(virus -> String.format("%s (%s integrations detected)", virus.type(), virus.integrations()))
+                .collect(Collectors.joining(", "));
+        return Optional.of(fusions.isEmpty() ? Formats.VALUE_NONE : fusions);
+    }
+
+    @NotNull
+    private Optional<String> actionableEventsWithoutHighDriverMutation() {
+        Set<String> eventsWithActinTrials = trials.stream()
+                .filter(EvaluatedTrial::isPotentiallyEligible)
+                .filter(EvaluatedTrial::isOpen)
+                .flatMap(trial -> trial.molecularEvents().stream())
+                .collect(Collectors.toSet());
+
+        Stream<? extends Driver> nonDisruptionDrivers = Stream.of(molecular.drivers().variants(),
+                        molecular.drivers().copyNumbers(),
+                        molecular.drivers().fusions(),
+                        molecular.drivers().homozygousDisruptions(),
+                        molecular.drivers().viruses())
+                .flatMap(Collection::stream)
+                .filter(driver -> driver.driverLikelihood() != DriverLikelihood.HIGH);
+
+        String events = Stream.concat(nonDisruptionDrivers, molecular.drivers().disruptions().stream())
+                .filter(driver -> !driver.evidence().externalEligibleTrials().isEmpty() || eventsWithActinTrials.contains(driver.event()))
+                .map(Driver::event)
+                .collect(Collectors.joining(", "));
```

Comment:
Just a thought, but I guess we also always want to display medium/low evidence that has associated standard-of-care evidence. I suspect Nina didn't think of this situation because it is incredibly unlikely that a variant has standard-of-care evidence and is not a HIGH driver.